hyp3_gamma/metadata/templates/rtc/product.xml.j2

```diff
@@ -9,7 +9,7 @@
 Image Analysis window. To learn more about speckle filtering options, refer to &lt;/SPAN&gt;&lt;A href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/speckle-function.htm" target="_blank" &gt;&lt;SPAN&gt;http://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/speckle-function.htm&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;, and to learn how apply a function (such as a speckle filter) to a raster, refer to &lt;/SPAN&gt;&lt;A href="http://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/editing-functions-on-a-raster-dataset.htm" target="_blank" &gt;&lt;SPAN&gt;http://desktop.arcgis.com/en/arcmap/latest/manage-data/raster-and-images/editing-functions-on-a-raster-dataset.htm&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;For areas where there is not a publicly-available digital elevation model (i.e. sea ice), geocoded products without terrain correction are also available from &lt;/SPAN&gt;&lt;A href="http://hyp3.asf.alaska.edu" target="_blank" &gt;&lt;SPAN&gt;http://hyp3.asf.alaska.edu&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Refer to ASF's &lt;/SPAN&gt;&lt;A href="https://hyp3-docs.asf.alaska.edu/guides/rtc_product_guide/" target="_blank" &gt;&lt;SPAN&gt;RTC Product Guide&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt; for more information about On Demand RTC products.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;The Sentinel-1 mission is designed to be a two-satellite constellation. Sentinel-1A was launched April 3, 2014, and remains active. Sentinel-1B was launched April 25, 2016, and was active until December 23, 2021. The satellites each have a 12-day repeat cycle and use the same orbit pattern, but are offset 180 degrees to allow repeat passes every 6 days while both are active.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Since the mission ended for S1B, there are some significant gaps in coverage. A reduced acquisition plan will be in place until Sentinel-1C is launched (expected in 2023) to replace Sentinel-1B. Visit &lt;/SPAN&gt;&lt;A href="https://hyp3-docs.asf.alaska.edu/sentinel1/" target="_blank" &gt;&lt;SPAN&gt;https://hyp3-docs.asf.alaska.edu/sentinel1&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt; for more information.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;More information about the Sentinel-1 mission is available at &lt;/SPAN&gt;&lt;A href="https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1" target="_blank" &gt;&lt;SPAN&gt;https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;P&gt;&lt;SPAN&gt;Additional information about Sentinel-1 data, imagery, tools and applications is available at &lt;/SPAN&gt;&lt;A href="https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1/" target="_blank" &gt;&lt;SPAN&gt;https://asf.alaska.edu/data-sets/sar-data-sets/sentinel-1&lt;/SPAN&gt;&lt;/A&gt;&lt;SPAN&gt;.&lt;/SPAN&gt;&lt;/P&gt;&lt;/DIV&gt;&lt;/DIV&gt;&lt;/DIV&gt;
```

Comment:
We may want to move the exclusionary language into the dem-cop.xml.j2 file instead, as this will not be relevant to products generated with the legacy DEMs. While that legacy DEM capability may go away, it's probably best to keep it specific to the COP DEM, anyway, rather than in the main template.

Reply:
This could be done using a "COP30_exemption" field.

Reply:
I have a [PR open to add this functionality](https://github.com/ASFHyP3/hyp3-gamma/pull/496). There may need to be some tweaking of the if/else statement in the macros file, but this provides the conceptual framework for my suggestion.